src/shared/bg_parse.cpp

```diff
@@ -803,7 +803,7 @@ void BG_ParseBuildableAttributeFile( const char *filename, buildableAttributes_t
 	else if ( !( defined & BUILDTIME) ) { token = "buildTime"; }
 	else if ( !( defined & NORMAL) ) { token = "minNormal"; }
 
-	if ( strlen( token ) > 0 )
+	if ( !Util::is_empty(token) )
```

Comment:
You could just do `if (*token)` which is a common idiom in the codebase. Also please rebase when fixing this since we prefer not to have any commits which fail to compile.

Reply:
Done